src/util/graph.h

```diff
@@ -458,6 +458,34 @@ void grapht<N>::visit_reachable(node_indext src)
     nodes[index].visited = true;
 }
 
+/// Add to `set` nodes that are reachable from `set`
+/// \param set: set of source nodes, must be a container with an
+///   `insert(const value_type&)` method.
+/// \param for_each_successor: function which given a node `n` and a function
+///   `f`, applies `f` on all successors of `n`.
```

Comment:
Could this comment please be a bit more technical and actually say which graph traversal algorithm is used (I think it's depth-first search), and explain what the idea behind this function is?

Reply:
I updated the comment